Specs/Scene/GroundPrimitiveSpec.js

```diff
@@ -499,13 +499,13 @@ defineSuite([
 
             largeScene.groundPrimitives.add(largeSceneDepthPrimitive);
             expect(largeScene).toRenderAndCall(function(rgba) {
-                expect(rgba.slice(0, 4)).not.toEqual([0, 0, 0, 255]);
+                expect(Array.prototype.slice.call(rgba, 0, 4)).not.toEqual([0, 0, 0, 255]);
```

Comment:
Cesium has an `arraySlice` utility function specifically because IE11 doesn't support it, so we should use that here instead.

Reply:
👍 thanks, I forgot about that